### ISSUE TITLE
Some questions about system performance

### ISSUE THREAD

**Issue by SakuraMemoryKnight** [laboshinl/loam_velodyne#155]
I read some of the papers on slam, and I found that in these papers the performance of slam was usually evaluated with the average relative translation （%） and rotation （deg/m）, It's just like the kitti list shows：
![77239946-483b8b00-6c1b-11ea-8c01-f2276333e6ae](https://user-images.githubusercontent.com/52240598/78528409-e3368680-7811-11ea-8159-d229402b534a.PNG)
How to get this result?
In fact, I don't quite understand the meaning of these two evaluation indexes, so I don't know what the formula is for this parameter.
I have used evo to test the slam performance results, and can get ape and rpe, but these are not the above two evaluation parameters.
I just want to know what their formula is. Is it the absolute error of the final frame divided by the total distance？

**Comment by RuanJY** [comment]
Just look at the text behind this table, there are information you need. 
"our evaluation computes translational and rotational errors for all possible subsequences of length (100,...,800) meters. The evaluation table below ranks methods according to the average of those values, where errors are measured in percent (for translation) and in degrees per meter (for rotation). A more detailed comparison for different trajectory lengths and driving speeds can be found in the plots underneath"

**Comment by lgm051** [comment]
> I read some of the papers on slam, and I found that in these papers the performance of slam was usually evaluated with the average relative translation （%） and rotation （deg/m）, It's just like the kitti list shows： ![77239946-483b8b00-6c1b-11ea-8c01-f2276333e6ae](https://user-images.githubusercontent.com/52240598/78528409-e3368680-7811-11ea-8159-d229402b534a.PNG) How to get this result? In fact, I don't quite understand the meaning of these two evaluation indexes, so I don't know what the formula is for this parameter. I have used evo to test the slam performance results, and can get ape and rpe, but these are not the above two evaluation parameters. I just want to know what their formula is. Is it the absolute error of the final frame divided by the total distance？

Hi, brother, do you know how to use Evo to measure these two parameters of Kitti? Can you give me some advises?

**Comment by RuanJY** [comment]
> 

Read their README. btw, I use this one: https://github.com/LeoQLi/KITTI_odometry_evaluation_tool, it works ok.

**Comment by SakuraMemoryKnight** [comment]
> > I read some of the papers on slam, and I found that in these papers the performance of slam was usually evaluated with the average relative translation （%） and rotation （deg/m）, It's just like the kitti list shows： ![77239946-483b8b00-6c1b-11ea-8c01-f2276333e6ae](https://user-images.githubusercontent.com/52240598/78528409-e3368680-7811-11ea-8159-d229402b534a.PNG) How to get this result? In fact, I don't quite understand the meaning of these two evaluation indexes, so I don't know what the formula is for this parameter. I have used evo to test the slam performance results, and can get ape and rpe, but these are not the above two evaluation parameters. I just want to know what their formula is. Is it the absolute error of the final frame divided by the total distance？
> 
> Hi, brother, do you know how to use Evo to measure these two parameters of Kitti? Can you give me some advises?

No, I just get rpe,ape,rte,ate with the EVO, but I wrote and calculated the average translation error and rotation error of the first 800 meters according to Kitti's instructions.
